thirdparty/README.md

```diff
@@ -0,0 +1,23 @@
+# What is `thirdparty`?
+
+This directory contains the files that are copied from 3rd-party projects and modified to fit kpt requirements.
+
+# What is in `thirdparty`?
+
+- `kyaml`: Files copied from [kyaml] library
```

Comment:
Where are you recording the commit ID for these?

Reply:
I would suggest that we should copy the changes from a known stable version of kyaml and record it somewhere.

Reply:
Updated readme. `kyaml` is copied from v0.10.15 and `cmd/config` is from v0.9.7. They are both latest versions.